RELEASING.md

```diff
@@ -0,0 +1,60 @@
+# Releasing Federation
+
+This is a quick and opinionated set of commands for building and releasing a new version of the packages in this monorepo.
+
+#### Start a release PR
+
+1. `FEDERATION_RELEASE_VERSION=X.Y.Z` (e.g. `2.0.1` or `2.0.0-beta.3` or `2.0.0-preview.99`)
+1. `git fetch origin main:main` to fetch the latest `main` branch
+1. `git checkout -b "release-$FEDERATION_RELEASE_VERSION" main`
+1. Update `CHANGELOG.md` in each repo ([more info](#changelogs))
+1. `git add ':/*CHANGELOG.md'` to add only changelogs to the commit
+1. `git commit --message "Update CHANGELOGs for v$FEDERATION_RELEASE_VERSION release"`
+1. `git push -u origin "release-$FEDERATION_RELEASE_VERSION"`
+1. `open "https://github.com/apollographql/federation/compare/main...release-$FEDERATION_RELEASE_VERSION?quick_pull=1&title=Release+$FEDERATION_RELEASE_VERSION&template=APOLLO_RELEASE_TEMPLATE.md"` to open PR in Github
+
+#### Bump version and tag release
+
+1. `npx lerna version --no-push --force-publish=\* "$FEDERATION_RELEASE_VERSION"` ([more info](#lerna-version))
+    1. `git show --pretty="" --name-only HEAD` to see the files in the commit
+    1. `git --no-pager tag --points-at HEAD` to see the tags that were created
+1. `git push --follow-tags origin` to push the version bumps & tags created by lerna in the previous step
+1. `APOLLO_DIST_TAG=next npm run release:start-ci-publish` ([more info](#publishing))
+1. `open https://app.circleci.com/pipelines/github/apollographql/federation?filter=mine` to approve publishing to NPM
+    1. There will also be a message posted to #team-atlas in slack that has a link to the approval job
+1. `npm view @apollo/subgraph versions` to ensure the version that was just published shows up in the NPM registry
```

Comment:
This command is helpful but maybe overly specific? It's at least pasteable and not hard to figure out how to validate the rest of them, so I think it's fine but I'm on the fence about it.

`npm view @apollo/subgraph@next` might also be an interesting variation with a similar result.

Reply:
Are you suggesting taking it out? Or adding the other packages in there to also check? Or something else? I agree that using the `@next` dist tag would be a good check.

Doing _something_ here was helpful for me as a very new deployer to make sure it did what I wanted.

Reply:
Sorry for the ambiguity, I'm not sure I have a preference. I agree something should be here, I just dislike all the options equally so I'd be equally disappointed to see it:
* as-is
* `npm view @apollo/<package-name>@next`
* with all of them listed

I think the pasteability aspect might be worth preserving which would rule out my second option.

Reply:
All roads lead to Trevor's sadness, noted 😸

Reply:
Added a few options, to ensure your complete misery